TACC/Core-CMS
# FP-1416: Callout Image Resize w/ JS not CSS

### Overview

Replace image resize JS solution for Callout Plugin with CSS solution.

### Issues

- **[FP-1416: CMS: Callout Image Resize to Use JS not CSS](https://jira.tacc.utexas.edu/browse/FP-1416)**
- GH-327
- GH-320

### Screenshots

| CSS Test | App Test |
| - | - |
| ![FP-1416 CSS Proof](https://user-images.githubusercontent.com/62723358/146270686-8687cf56-ea2f-4f6c-894e-1573ba5ff62e.png) | ![FP-1416 App Proof](https://user-images.githubusercontent.com/62723358/146270682-8edcb46f-52db-4946-aeb5-b5d287ee03c0.png) |

### Testing

0. (If testing locally) Checkout branch, build and collect static files, run migrations.
1. Create instance of Callout Plugin.
2. Confirm Callout plugin now does **not** mention image resize in "Advanced Settings". 
2. Add an instance of an Image Plugin within it.
3. Test that image resize works:
    - Image shorter than text is automatically resized to match text height.
    - Image taller than text is automatically resized to match text height.
4. Check chat styles have not changed.
